### PR TITLE
Varvara/feature/google apple auth

### PR DESCRIPTION
**Pull request by varvarashcherbyna7**

### CODE REVIEW

**Review by BohdanSizomin — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
 node_modules
+.env
+.eslintcache
+build
```
Comment (kipod) [inline .gitignore:4]: /build/

File: src/components/Home/Home.tsx [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    gapi.load('client:auth2', start);
+    const authUser = localStorage.getItem('user');
+    if (authUser) setUser(JSON.parse(authUser ?? ''));
+  }, []);
+
+  // get accessToken
+  // const accessToken = gapi.auth.getToken().access_token;
+  console.log('user ', user);
+
+  const onSuccess = () => {
```
Comment (kipod) [inline src/components/Home/Home.tsx]: onLogoutSuccess

File: src/components/Home/Home.test.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+//       <SignIn/>
+//     );
+
+//     const heading = screen.getByRole('heading', {
+//       name: /welcome to next\.js!/i,
+//     });
+
+//     expect(heading).toBeInTheDocument();
+//   });
+// });
```
Comment (kipod) [inline src/components/Home/Home.test.tsx:29]: do we really need all these commented lines in the code?

File: src/components/SingIn/SignIn.test.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+//       <SignIn/>
+//     );
+
+//     const heading = screen.getByRole('heading', {
+//       name: /welcome to next\.js!/i,
+//     });
+
+//     expect(heading).toBeInTheDocument();
+//   });
+// });
```
Comment (kipod) [inline src/components/SingIn/SignIn.test.tsx:29]: pls remove commented code

File: src/components/common/Loader/Loader.test.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+//       <SignIn/>
+//     );
+
+//     const heading = screen.getByRole('heading', {
+//       name: /welcome to next\.js!/i,
+//     });
+
+//     expect(heading).toBeInTheDocument();
+//   });
+// });
```
Comment (kipod) [inline src/components/common/Loader/Loader.test.tsx:29]: copy-paste???

**Review by kipod — APPROVED** [review]

File: src/components/SingIn/SignIn.test.tsx (hunk 4 above)
Comment (varvarashcherbyna7) [reply src/components/SingIn/SignIn.test.tsx:29]: ok)

File: src/components/Home/Home.test.tsx (hunk 3 above)
Comment (varvarashcherbyna7) [reply src/components/Home/Home.test.tsx:29]: I  will remove this

File: .gitignore (hunk 1 above)
Comment (varvarashcherbyna7) [reply .gitignore:4]: Yes, we don't need a build folder on git

File: src/components/Home/Home.tsx (hunk 2 above)
Comment (varvarashcherbyna7) [reply src/components/Home/Home.tsx]: ок)

**Pull request merged**